docs/buildah-add.md

```diff
@@ -27,6 +27,15 @@ BUILDAH\_HISTORY environment variable. `export BUILDAH_HISTORY=true`
 
 Sets the user and group ownership of the destination content.
 
+**--contextdir**
+
+Build context directory. Specifying a context directory, causes Buildah to
+chroot into a the context directory. This means copying files pointed at by symbolic links outside of the chroot will fail.
+
+**--ignorefile**
+
+Path to alternative .dockerignore file. Requires --contextdir be specified.
```

Comment:
If this is a relative path, where should it be relative to?

Reply:
I am not sure what the question is?  Whether or not the file is relative does not mapper.  We are just opening and reading the file, and setting the options.Excludes.

Reply:
Good point Nalin, might be best to say fully specified directory or some such.